mifosng-android/src/main/java/com/mifos/mifosxdroid/online/collectionsheetindividual/IndividualCollectionSheetFragment.java

```diff
@@ -0,0 +1,413 @@
+    private int mStaffId;
+    private IndividualCollectionSheetAdapter sheetsAdapter;
+
+    private IndividualCollectionSheetPayload mPayload;
+
+    private String actualDisbursementDate;
+    private String transactionDate;
+
+    public IndividualCollectionSheetFragment() {
+    }
+
+    public static IndividualCollectionSheetFragment newInstance() {
+
+        Bundle args = new Bundle();
+
+        IndividualCollectionSheetFragment fragment = new IndividualCollectionSheetFragment();
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        ((MifosBaseActivity) getActivity()).getActivityComponent().inject(this);
+        if (getArguments() != null) {
+            
+        }
+
+        if (savedInstanceState != null) {
+            mSheet = (IndividualCollectionSheet) savedInstanceState.get(
+                    Constants.EXTRA_COLLECTION_INDIVIDUAL);
+            showCollectionSheetViews(mSheet);
+        }
+
+        setHasOptionsMenu(true);
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater,
+                             @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        rootView = inflater.inflate(R.layout.fragment_individual_recycler, container, false);
+        ButterKnife.bind(this, rootView);
+        setToolbarTitle(getStringMessage(R.string.individual_collection_sheet));
+        presenter.attachView(this);
+
+        setUpUi();
+        return rootView;
+    }
+
+    @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        inflater.inflate(R.menu.menu_individual_collectionsheet, menu);
+        super.onCreateOptionsMenu(menu, inflater);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.item_submit_sheet:
+                submitSheet();
+                return true;
+
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    void setRepaymentDate() {
+        mDatePicker = MFDatePicker.newInsance(this);
+        String date = DateHelper.
+                getDateAsStringUsedForCollectionSheetPayload(MFDatePicker.getDatePickedAsString());
+        tvRepaymentDate.setText(date.replace('-', ' '));
+        transactionDate = date.replace('-', ' ');
+        actualDisbursementDate = transactionDate;
+    }
+
+    private void prepareRequestPayload() {
+        requestPayload = new RequestCollectionSheetPayload();
+        requestPayload.setOfficeId(mOfficeId);
+        requestPayload.setStaffId(mStaffId);
+        requestPayload.setTransactionDate(tvRepaymentDate.getText().toString());
+
+
+        /* Below 3 lines for Testing only */
+
+        /*requestPayload.setOfficeId(1);
+        requestPayload.setStaffId(12);
+        requestPayload.setTransactionDate("06 July 2017");*/
```

Comment:
@tarun0 this comment is not making any sense, if you think, this is useful for another developer so add as a comment on method and add these line with proper example.